microsoft-365/enterprise/external-domain-name-system-records.md

```diff
@@ -85,13 +85,10 @@ These DNS records apply to Teams, Skype for Business Online or both as indicated
-|**SRV** <br/> **(Teams and Skype for Business Online)**|Required by Skype for Business Windows Desktop client and Skype for Business phones for sign-in. It may be needed by Teams-only tenants that use Skype for Business Online phones for Teams and must point to online edge servers (e.g. _sip.online.lync.com_). <br/>It is needed by hybrid tenants to support their Windows Desktop clients and phones that sign in to on-premises deployments (e.g. _sip.\<domain>_).|**Target:** _sip._tls.\<domain>|
-|**CNAME** <br/> **(Teams and Skype for Business Online)**|Required by the Skype for Business desktop client for Windows, Mac, and web clients as well as Skype Meeting Application (SMA) to sign in. <br/>Also used by PowerShell cmdlets that still use Skype for Business Online infrastructure for management. Therefore, it is also needed for both Teams-only and hybrid tenants.|**Target:**  lyncdiscover.\<domain>|
-|**SRV** <br/> **(Skype for Business Online)**|Required by Skype for Business to coordinate the flow of information between Lync clients.|**Service:** sip <br/> **Protocol:** TLS <br/> **Priority:** 100 <br/> **Weight:** 1 <br/> **Port:** 443 <br/> **Target:** sipdir.online.lync.com|
-|**CNAME** <br/> **(Skype for Business Online)**|Required by the Lync desktop client to locate the Skype for Business Online service and sign in.|**Alias:** sip <br/> **Target:** sipdir.online.lync.com|
-|**CNAME** <br/> **(Skype for Business Online)**|Required by the Lync mobile client to help find the Skype for Business Online service and sign in.|**Alias:** lyncdiscover <br/> **Target:** webdir.online.lync.com|
+|**SRV** <br/> **(Teams)**|Allows your Office 365 domain to share instant messaging (IM) features with external clients by enabling SIP federation.|**Service:** sipfederationtls <br/> **Protocol:** TCP <br/> **Priority:** 100 <br/> **Weight:** 1 <br/> **Port:** 5061 <br/> **Target:** sipfed.online.lync.com <br/> **Note:** If the firewall or proxy server blocks SRV lookups on an external DNS, you should add this record to the internal DNS record. |
+|**SRV** <br/> **(Teams)**| It may be needed by Teams-only tenants that use Skype for Business Online phones for Teams and must point to online edge servers (e.g. _sip.online.lync.com_). <br/>It is needed by hybrid tenants to support their Windows Desktop clients and phones that sign in to on-premises deployments (e.g. _sip.\<domain>_).|**Target:** _sip._tls.\<domain>|
```

Comment:
```suggestion
|**SRV** <br/> **(Teams)**| It may be needed by Teams-only tenants that use Skype for Business Online phones for Teams and must point to online edge servers (such as _sip.online.lync.com_). <br/>It is needed by hybrid tenants to support their Windows Desktop clients and phones that sign in to on-premises deployments (such as _sip.\<domain>_).|**Target:** _sip._tls.\<domain>|
```